test/tests/functional/pbs_hook_execjob_end.py

```diff
@@ -160,3 +163,52 @@ def test_execjob_end_multi_job(self):
                                        max_attempts=10, interval=2)
```

Comment:
endjob hook for job1 should end within 10s when it started. I do not understand why are you checking it against the start time of job2's end job hook?

Reply:
since we are trying to ensure execjob_end log message of jid2 is logged between the execjob_end started for jid1 log message and execjob_end execution ended fir jid1 log message.